commons/zenoh-protocol/src/network/oam.rs

```diff
@@ -12,15 +12,22 @@
 //   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
 //
 use crate::common::ZExtBody;
-
-pub type OamId = u16;
+use crate::transport::oam::OamId;
```

Comment:
This is wrong. We are in the network layer. Do NOT import things from the transport layer. Revert.